src/OrchardCore.Modules/OrchardCore.Admin/AdminMenu.cs

```diff
@@ -13,7 +13,7 @@ public sealed class AdminMenu : AdminNavigationProvider
         { "groupId", AdminSiteSettingsDisplayDriver.GroupId },
     };
 
-    internal readonly IStringLocalizer S;
+    private readonly IStringLocalizer S;
```

Comment:
Does the project show you suggestion under the variable S? The reason we used internal is because we have a rule that say private fields should start with `_`.

Reply:
This is another story. :) I know @sebastienros likes this convention, but nowadays, the PO Extractor tool can discover the localizers by supplying names like `S`, `H`, or anything else

Reply:
Well I won't not convert all these properties to private if we are going to get the hint suggesting to rename the field.

Reply:
`internal` doesn't hurt here in IMHO. Whereas `private` will force a name violation, as @MikeAlhayek said. I think we should not do this at the moment.

Reply:
IMHO using `internal` is wrong right now, but if the intent is to hide name violation, I suggest fixing the main reason, which is using `S` & `H`

@sebastienros I think there's no reason nowadays to use `S` & `H` fields. I modified the PO Extractor tool to be able to extract the resources by supplying custom names

We should use `T`, `S`, and `H` for properties,s not fields

Reply:
Yes, with a changed naming convention, we can make this look better. Maybe we can simply allow `_s, _t, _h` even though this doesn't look that pretty.

Reply:
What I'm suggesting is to have `S`, `H`, and `T` if they are applicable to be properties such as `RazorPageBase`, which has `T`; otherwise, we can use `_stringLocalizer`, `_htmlLocalizer`

It would be nice if you could discuss that with Seb in the Triage meeting, if you need me to prepare a demo that allows you extract such localization string I can do

Reply:
@sebastienros, I never mind closing this PR, but we need to think about naming `S` & `H` again